e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -248,6 +248,66 @@ export function createTextCard({
   };
 }
 
+export function createHeadingCardDTO({
```

Comment:
I added "DTO" to the name as similar helper functions existed that used the GUI to create the cards

Reply:
We don't use DTOs suffixes. If it was used somewhere it wasn't intended

Reply:
Do we have a naming convention/strategy to distinguish creating things with the UI:
https://github.com/metabase/metabase/blob/b55a4070e4b271baa8ad89ecdfec6722e419eaf9/e2e/support/helpers/e2e-dashboard-helpers.js#L192-L194

from creating just the data to send to the API:

https://github.com/metabase/metabase/blob/b55a4070e4b271baa8ad89ecdfec6722e419eaf9/e2e/support/helpers/e2e-dashboard-helpers.js#L280-L309

?

Reply:
No, we don't

Reply:
Actually, we have. We use `details` prefix sometimes.